src/jarabe/model/friends.py

```diff
@@ -156,9 +156,10 @@ def save(self):
         cp = ConfigParser()
 
         for friend in self:
-            section = friend.get_key()
-            cp.add_section(section)
-            cp.set(section, 'nick', friend.get_nick())
+            section = str(friend.get_key())
```

Comment:
@Saumya-Mishra9129 Can you explain the need for `str` function?

Reply:
> Can you explain the need for `str` function?

ConfigParser's set method takes 1st argument as string or raise TypeError.

Reply:
Great! Thanks. Is it possible to infer the type of the return of `.get_key()` as an additional message to the commit message so it can be useful for future contributors.

---------
The PR looks good

Reply:
You may mark this as resolved

Reply:
.get_key() returns a byte.